README.md

```diff
@@ -118,7 +121,15 @@ end
 ```
 
 ```rb
-# Your answer goes here...
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+    respond_to do |format|
+      format.html { render json: @posts }
```

Comment:
You don't want this inside the `format.html` portion. You want to direct the user to the proper view (e.g., `:index`)